modules/olm-subscriptions/chart/values.yaml

```diff
@@ -49,6 +50,7 @@ subscriptionConfig:
 
 
 bookkeeper:
+  channel: stable
```

Comment:
We don't need to specify channel for every sub, because there's a default value in `.Values.channel`.

Reply:
I wonder if no helm value `channel` is specified in `values.yaml` the terraform modules olm-subscriptions could render this value correctly? @ciiiii

Reply:
No, but it's in default values, so this won't happen

Reply:
Yes, component channel value has high priority. `.Values.channel` is the default value, we can overwrite it if we need to

Reply:
That's true, I just validated it.